src/Turbo/src/Bridge/Mercure/Broadcaster.php

```diff
@@ -68,11 +68,6 @@ public function broadcast(object $entity, string $action, array $options): void
         $topics = [];
 
         foreach ((array) ($options['topics'] ?? []) as $topic) {
-            if (!\is_string($topic)) {
-                $topics[] = $topic;
-                continue;
-            }
```

Comment:
I and PHPStan does not see how `$topic` can not be a string at this point.

Reply:
friendly ping @rskuipers, by any chance :)

Reply:
Just in case of, I will revert this modification and ignore it from PHPStan, we will remove it safely in 3.x